pkg/acsengine/engine_test.go

```diff
@@ -66,20 +51,44 @@ func TestExpected(t *testing.T) {
 			t.Error(e3.Error())
 			continue
 		}
+
+		armTemplate, params, certsGenerated, err := templateGenerator.GenerateTemplate(containerService)
+		if err != nil {
+			t.Error(fmt.Errorf("error in file %s: %s", tuple.APIModelFilename, err.Error()))
+			continue
+		}
+
+		ppArmTemplate, e1 := PrettyPrintArmTemplate(armTemplate)
```

Comment:
for ease of reading prefix ppArmTemplate and ppParams with "expected" or "target"